lib/improver/wind_downscaling.py

```diff
@@ -336,13 +337,16 @@ def calc_roughness_correction(self, hgrid, uold, mask):
         """Function to perform the roughness correction.
 
         Args:
-            hgrid: 3D or 1D np.array (float)
+            hgrid: 3D or 1D np.array (float32)
                 height above orography
-            uold: 3D np.array (float)
+            uold: 3D np.array (float32)
                 original velocities at hgrid.
+            mask: 2D np.array logical
```

Comment:
The correct format for docstrings within IMPROVER are documented within the [Code Style Guide](https://github.com/metoppv/improver/wiki/Code-Style-Guide) i.e.
```python
    name_of_variable (variable_type):
```
For example:
```python
    mask (bool):
```
We've adopted this style so that it can be interpreted by Sphinx to create the online documentation.
Could you update this, so that it matches this style? The sentence below could contain a mention that the input array will be 2D e.g.
```python
    """
    mask (bool):
    2D numpy array that is True for land-points, False for Sea and False for
    invalid z_0.
    """
```